src/Mod/TechDraw/TechDrawTools/TaskFillTemplateFields.py

```diff
@@ -520,17 +524,19 @@ def chkEachChkBox(self):
             self.button.setEnabled(False)
 
     def proceed(self):
+        transactionName = QtCore.QT_TRANSLATE_NOOP("Techdraw_FillTemplateFields", "Fill template fields")
```

Comment:
Noting that this was not being translated before, so though the string is new for translators, it won't represent a UI regression when merged.